### PR TITLE
Reword application renewal string

### PR DESCRIPTION
**Pull request by Shubh1815**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Reword application renewal string. As "app" more often refers to mobile application.

Original string: "Attempt to renew unapproved app #{pk} has been denied"
Changed string: "Attempt to renew unapproved application #{pk} has been denied"

## Rationale
[//]: # (Why is this change required? What problem does it solve?)

## Phabricator Ticket
[//]: # [T275042](https://phabricator.wikimedia.org/T275042)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: TWLight/applications/views.py [hunk 1]
```diff
@@ -1556,10 +1556,11 @@ def get_object(self):
             messages.add_message(
                 self.request,
                 messages.WARNING,
+                # fmt: off
```
Comment (suecarmol) [inline TWLight/applications/views.py:1559]: Please remove the `# fmt: off` and `# fmt: on` comments.

**Review by suecarmol — CHANGES_REQUESTED** [review]
Thank you for your contribution. This needs some changes before we can go ahead and merge this.

File: TWLight/applications/views.py (hunk 1 above)
Comment (Shubh1815) [reply TWLight/applications/views.py:1559]:
```
messages.add_message(
                self.request,
                messages.WARNING,
                # Translators: This message is displayed when an attempt by a user to renew an application has been denied for some reason.
                _("Attempt to renew unapproved application #{pk} has been denied")
                .format(pk=app.pk),
)
```
If I keep it like this, then the lint test fails and tells to reformat the above block of code into this:-

```
messages.add_message(
                self.request,
                messages.WARNING,
                # Translators: This message is displayed when an attempt by a user to renew an application has been denied for some reason.
                _(
                    "Attempt to renew unapproved application #{pk} has been denied"
                ).format(pk=app.pk),
            )
```

And if I keep the reformatted block of code and then I run test then its show this error.
```
ERROR: ugettext messages should't be preceded by a newline
TWLight/applications/views.py:1561
                    "Attempt to renew unapproved application #{pk} has been denied"

```

I checked some different messages in that file and some of them were using those comments like this one.
[TWLight/applications/views.py#L1530](https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/applications/views.py#L1530)

Comment (suecarmol) [reply TWLight/applications/views.py:1559]: Got it! I think then you should keep the black formatting comments, then. Thanks for letting me know

**Review by suecarmol — APPROVED** [review]
Thanks for changing this string. Moving on to merge!

**Pull request merged**